src/test/java/core/basesyntax/service/FileReaderTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.impl.FileReaderImpl;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderTest {
+
+    private static final String READER_FILE_PATH = "src/test/resources"
+            + "/fileReaderResources/input_OK.csv";
+    private static final String NOT_EXISTING_FILE_PATH = "src/test/resources"
+            + "/fileReaderResources/nevermind.csv";
+    private final FileReader fileReader = new FileReaderImpl();
```

Comment:
```suggestion
    private final FileReader fileReader;
    
        @BeforeClass
    public static void beforeClass() {
       fileReader = new FileReaderImpl();
    }
```
You can move initialization of this field, and all the fields of this kind in the Test classes in special method @BeforeClass